web/models/class/class.ex

```diff
@@ -31,6 +32,13 @@ defmodule CoursePlanner.Class do
     |> validate_required([:offered_course_id, :date, :starting_at, :finishes_at, :status])
   end
 
+  def changeset(struct, params, :fill) do
+    struct
+    |> cast(params, [:course_id])
```

Comment:
this i wrote before you change to offered_course_id. I have to change it to "offered_course_id"